news/2016-01-29-WildFly10-Released.adoc

```diff
@@ -42,7 +42,7 @@ For more details see the following blog post: link:http://wildfly.org/news/2015/
 
 HA Singleton Deployments
 ------------------------
-WildFly 10 adds the ability to deploy a given application as a "singleton deployment". This is a new implementation of a feature that existed in AS 6.0 and earlier. When deployed to a group of clustered servers, a singleton deployment will only deploy on a single node at any given time. If the node on which the deployment is active stops or fails, the deployment will automatically start on another node. The policies for controlling HA singleton behavior are managed by a new "singleton" subsystem.  A deployment may either specify a specific singleton policy or use the default subsystem policy. A deployment identifies itself as singleton deployment via a +/META-INF/singleton-deployment.xml+ deployment descriptor (the schema for which link:https://github.com/wildfly/wildfly/blob/master/clustering/singleton/extension/src/main/resources/schema/singleton-deployment_1_0.xsd[can be found on GitHub]), which is most easily applied to an existing deployment as a deployment overlay. Alternatively, the requisite singleton configuration can be embedded within an existing +jboss-all.xml+.
+WildFly 10 adds the ability to deploy a given application as a "singleton deployment". This is a new implementation of a feature that existed in AS 6.0 and earlier. When deployed to a group of clustered servers, a singleton deployment will only deploy on a single node at any given time. If the node on which the deployment is active stops or fails, the deployment will automatically start on another node. The policies for controlling HA singleton behavior are managed by a new "singleton" subsystem.  A deployment may either specify a specific singleton policy or use the default subsystem policy. A deployment identifies itself as singleton deployment via a +/META-INF/singleton-deployment.xml+ deployment descriptor (the schema for which link:https://github.com/wildfly/wildfly/blob/10.0.0.Final/clustering/singleton/extension/src/main/resources/schema/singleton-deployment_1_0.xsd[can be found on GitHub]), which is most easily applied to an existing deployment as a deployment overlay. Alternatively, the requisite singleton configuration can be embedded within an existing +jboss-all.xml+.
```

Comment:
Github doesn't seem to show the exact change on the line: the change is to link to the xsd directly in the WF10 tag instead of master.